testimony/__init__.py

```diff
@@ -63,274 +195,283 @@ def main(report, paths, json_output, nocolor):
-            else:
-                col = CLR_ERR
-            print colored(
-                PRINT_NO_MINIMUM_DOC_TC,
-                attrs=['bold']) % colored(result.no_minimal_docstring, col)
-        # Print number of test cases affected by bugs and also the list of bugs
-        if report == BUGS_REPORT:
-            print colored(
-                PRINT_TC_AFFECTED_BUGS, attrs=['bold']) % result.bugs
-            if len(result.bugs_list) > 0:
-                print colored("\nBug list:", attrs=['bold'])
-                for bug in result.bugs_list:
-                    print bug
-
-
-def print_json_output(report, results):
-    """Prints the report output in JSON format"""
-
-    if report == PRINT_REPORT:
-        output = [result.paths for result in results]
-    elif report == SUMMARY_REPORT:
-        output = [get_summary_result(result) for result in results]
-    elif report == VALIDATE_REPORT:
-        output = [result.__dict__ for result in results]
-    elif report == BUGS_REPORT:
-        output = [{'bugs': result.bugs,
-                   'bugs_list': result.bugs_list} for result in results]
-    elif report == MANUAL_REPORT or report == AUTO_REPORT:
-        for result in results:
-            for path in result.paths:
-                for filepath in path['files']:
-                    docstrings = []
-                    for docstring in filepath['docstrings']:
-                        status_tag = False
-                        for line in docstring:
-                            tag = line.split(' ', 1)[0].lower()
-                            if DOC_TAGS[6] in tag:
-                                status_tag = True
-                        if ((report == AUTO_REPORT and not status_tag) or
-                                (report == MANUAL_REPORT and status_tag)):
-                            docstrings.append(docstring)
-                    filepath['docstrings'] = docstrings
-        output = [result.paths for result in results]
-    else:
-        # Will not get here because argparse validation, but just to make sure
-        raise Exception('Report %s not available' % report)
-
-    print json.dumps(output)
-
-
-def get_docstrings(report, path, result):
-    """Function to read docstrings from test_*** methods for a given file"""
-    return_list = []
-    with open(path) as handler:
-        test_methods = [
-            node for node in ast.walk(ast.parse(handler.read()))
-            if isinstance(node, ast.FunctionDef) and
-            node.name.startswith('test_')
-        ]
-    for test_method in test_methods:
-        docstring = ast.get_docstring(test_method)
-        item_list = []
-        if docstring is None:
-            if (report == PRINT_REPORT or report == VALIDATE_REPORT):
-                item_list.append('%s: %s' % (
-                    test_method.name, colored(PRINT_DOC_MISSING, CLR_ERR)))
-            return_list.append(item_list)
-            result.no_docstring += 1
+    if report == 'summary':
```

Comment:
I used constant names earlier for report names like 'summary', 'print', 'validate_docstring', 'bugs', 'manual', 'auto' inorder to easily rename these in future.  Can you follow the same approach here please?